### ISSUE TITLE
1MB/s Cap for Premium Member

### ISSUE THREAD

**Issue by medievalmace** [ModOrganizer2/modorganizer#632]
I'm a premium member on the nexus and MO2 doesn't seem to want to download more than 1MB/s, nor does it want to pick up any other servers.

MO2 Version: 2.1.6

Windows 10 OS

I've looked at other similar topics, but the whole "clear cache and restart" thing doesn't exactly work for me. The download speed was working fine with Vortex before I re-associated .nxm files with MO.

**Comment by Al12rs** [comment]
We would need the inteface log for these kinds of things, you can get it like this:
```To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder". You should close MO2 before sending or examining the logs as they are finished writing only after the program is closed. 
```
Can you confirm that Mo2 successfully recognizes you as a premium user, by checking if on the application bar (the one where there is close X expand/ minimize) your nexus name is followed by a `*` ?

**Comment by jy4git** [comment]
I was having difficulties with MO2 and Nexus integration for downloading files. MO2 would first tell me login failed then I canceled the retry login. After that I was not told again that the Login failed and MO2 would not find known servers. I discovered that MO2 wanted a different username from what I use as a username logging into nexus in a web browser like chrome. For example, my username for Nexus in a browser is my email address but in M02 my nexus username is my Nexus screen name people see when I post a comment on a mod.
